webui/react/src/utils/shortcut.ts

```diff
@@ -1,17 +1,31 @@
-export interface KeyboardShortcut {
-  ctrl: boolean;
-  meta: boolean;
-  alt: boolean;
-  shift: boolean;
-  key?: string;
-}
+import * as t from 'io-ts';
+
+import { optional } from 'ioTypes';
+
+export const KeyboardShortcut = t.type({
+  alt: optional(t.boolean),
```

Comment:
I don't understand why we would want any of these fields to be optional. It seems to me that a modifier key cvan only be pressed or not (boolean) and the key is required.